src/pages/BasketPage/BasketPage.scss

```
@@ -0,0 +1,136 @@
+.basket-page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 150px;
+  padding-bottom: 400px;
+
+  .basket-name {
+    font-size: 30px;
+    font-weight: bold;
+    padding-bottom: 70px;
+  }
+
+  .basket-flex {
+    display: flex;
+
+    .basket {
+      .basket-table {
+        display: flex;
+        justify-content: space-around;
+        width: 860px;
+        border: 1px solid black;
+        border-left: none;
+        border-right: none;
+
+        .basket-infor {
+          padding: 20px;
+        }
+
+        .basket-num {
+          position: relative;
+          left: 60px;
+          padding: 20px;
+        }
+
+        .basket-price {
+          position: relative;
+          left: 60px;
+          padding: 20px;
+        }
+      }
+
+      .basket-item {
+        display: flex;
+        justify-content: center;
+        justify-content: space-around;
+        padding-top: 20px;
+        width: 860px;
+        padding-bottom: 50px;
+        border-bottom: 1px solid gray;
+
+        .item-name {
+          padding: 30px;
+        }
+
+        .item-num,
+        .item-price {
+          padding: 30px;
+          position: relative;
+          left: 60px;
+        }
+      }
+    }
+
+    .basket-payment {
+      position: relative;
+      top: -51px;
+      margin-top: 51px;
+      margin-left: 39px;
+      padding-top: 30px;
+      padding-bottom: 30px;
+      border-top: 1px solid black;
+      width: 280px;
+      background-color: #f8f8f8;
+
+      .payment-title {
+        display: flex;
+        justify-content: space-around;
+        padding: 30px;
+
+        .payment-name,
+        .payment-money {
+          font-weight: bold;
+        }
+      }
+      .payment-expectation {
+        position: relative;
+        left: -28px;
+        text-align: end;
+
+        .expectation-name {
+          font-size: 13px;
+          font-weight: bold;
+          line-height: 24px;
+        }
+
+        .expectation-money {
+          margin-top: 2px;
+          font-size: 24px;
+          font-weight: bold;
+          line-height: 24px;
+          color: #d0021b;
+        }
+      }
+      .payment-button {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        .basket-order {
+          width: 80%;
+          height: 50px;
+          border: none;
+          color: white;
+          font-size: 15px;
+          font-weight: bold;
+          margin-top: 10px;
+          background-color: red;
+        }
+
+        .basket-out {
+          display: block;
+          width: 80%;
+          height: 50px;
+          border: none;
+          color: white;
+          font-size: 15px;
+          font-weight: bold;
+          margin-top: 10px;
+          background-color: gray;
+        }
```

```suggestion
        .basket-order {
          width: 80%;
          height: 50px;
          margin-top: 10px;
          border: none;
          background-color: red;
          color: white;
          font-size: 15px;
          font-weight: bold;

          &.out {
            background-color: gray;
            color: white;
          }
        }
```
중복되는 스타일링이 많아서 변경되는 부분만 스타일링을 주는 방향으로도 생각해 보셔도 좋을 것 같네요.